url78eec0e0b6_Orikuro_vic_tgz-pJ8-net_nexon_vindictus_itemcomparer_inport_StaticImportJ8/src/net/nexon/vindictus/itemcomparer/modell/Combo.java

```diff
@@ -0,0 +1,270 @@
+package net.nexon.vindictus.itemcomparer.modell;
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import net.nexon.vindictus.itemcomparer.modell.ext.Armor;
+import net.nexon.vindictus.itemcomparer.modell.ext.Gloves;
+import net.nexon.vindictus.itemcomparer.modell.ext.Helm;
+import net.nexon.vindictus.itemcomparer.modell.ext.Pants;
+import net.nexon.vindictus.itemcomparer.modell.ext.Shoes;
+
+public class Combo {
+
+	public static final String OUTPUT_HEAD = "id;def;matk;atk;sta;c_res;bal;crit;aspd;price;shoes;pants;gloves;armor;helm;";
+
+	private static DecimalFormat df = new DecimalFormat(",###.00");
+
+	@Override
+	public String toString() {
+		String price_norm = df.format(price);
+		int atk_norm = (int) atk;
+		return String.format("%s;%s;%s;%s;%s;%s;%s;%s;%s;%s;%s;%s;%s;%s;", def,
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format int: sta, &#39;calccrit&#39; expression, def, &#39;calcbal&#39; expression, matk, atk_norm (declared at line 23), c_res, &#39;calcaspd&#39; expression. This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 24. Consider formatting this data with &#39;%d&#39; instead.